<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblCpuUse">
         <property name="text">
          <string>Uso de CPU: ?%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCpuIdle">
         <property name="text">
          <string>CPU Ociosa: ?%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMemUse">
         <property name="text">
          <string>Uso de Ram: ?%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMemFree">
         <property name="text">
          <string>Ram Livre: ?%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblTotalProc">
         <property name="text">
          <string>Processos: ?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tblProcess">
      <property name="columnCount">
       <number>14</number>
      </property>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Usuário</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CPU%</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Memória (MB)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Threads</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Estado</string>
       </property>
      </column>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblUpdate">
      <property name="text">
       <string>Última atualização: 00:00:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
